ui/src/components/policy/PolicyRow.js

```diff
@@ -40,6 +46,28 @@ const TrStyled = styled.tr`
 `;
 
 const TrStyledPolicy = styled.tr`
+    box-sizing: border-box;
+    margin-top: 10px;
+    box-shadow: 0 1px 4px #d9d9d9;
+    box-shadow: ${(props) => {
```

Comment:
Each row should have a different border / shadow depending on the version location inside the expanded policy view (first - shadow on top, middle - almost no shadow, final - shadow in the bottom, single - top shadow and bottom shadow (when policy isn't expanded)